<launch>
<arg name="play_bag" default="true"/>
<param name="/use_sim_time" value="true" />

<group ns="robot1">
<arg name="robot_name" default="robot1"/>

<arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
<param name="model" value="$(arg model)"/>

<node pkg="gmm_map_python" type="exploration.py" name="exploration_robot1"  output="screen" required="true">
        <remap from="sampled_points" to="Downsample/sampled_points"/>
        <param name="baselink_frame" value="$(arg robot_name)/base_link"/>
        <param name="odom_frame" value="$(arg robot_name)/odom"/>
        <param name="map_frame" value="$(arg robot_name)/map"/>
        <param name="robot_id" value="1"/>
</node>
</group>

<group ns="robot2">
<arg name="robot_name" default="robot2"/>

<arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
<param name="model" value="$(arg model)"/>

<node pkg="gmm_map_python" type="exploration.py" name="exploration_robot2"  output="screen" required="true">
        <remap from="sampled_points" to="Downsample/sampled_points"/>
        <param name="baselink_frame" value="$(arg robot_name)/base_link"/>
        <param name="odom_frame" value="$(arg robot_name)/odom"/>
        <param name="map_frame" value="$(arg robot_name)/map"/>
        <param name="robot_id" value="2"/>
</node>
</group>


</launch>